config/app/config.yaml

```
@@ -5,3 +5,8 @@ sylius_ui:
                 commerce_weavers_sylius_also_bought_missing_configuration:
                     template: '@CommerceWeaversSyliusAlsoBoughtPlugin/_missing_configuration.html.twig'
                     priority: 40
+        sylius.admin.channel.form.second_column_content:
+            blocks:
+                commerce_weavers_sylius_also_bought_number_of_synchronised_products:
```

```suggestion
                commerce_weavers_sylius_also_bought_number_of_synchronized_products:
```

just because in different places, we already use the US version with `z`

---

same for twig tempalte file name etc.